--- Running scipy.fft.dst#34YY/dstYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'scipy.fft.dst#34YY'), ('notebook', 'dstYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.456s
user	0m1.735s
sys	0m6.933s
